src/components/MinimalCollapse.tsx

```diff
@@ -17,7 +22,9 @@ export function MinimalCollapse({
         background: 'transparent',
         ...style,
       }}
-      className="minimal ant-collapse-header-text-primary ant-collapse-header-p-0"
+      className={`minimal ant-collapse-header-text-primary ant-collapse-header-p-0 ${
+        light ? 'light' : ''
```

Comment:
whats the `light` class do? Is it possible to use the ThemeContext here instead of the prop I wonder?

Reply:
@tomquirk changes the font-weight actually, not a colour thing